web/docs/introduction/what-is-wasp.md

```diff
@@ -0,0 +1,174 @@
+---
+title: What is Wasp?
+---
+
+import ImgWithCaption from '../../blog/components/ImgWithCaption'
+
+We will give a brief overview of what Wasp is, how it works on a high level and when to use it.
+
+## Wasp is a tool to build modern web applications
+
+It is an opinionated way of building **full-stack web applications**. It takes care of all three
+major parts of a web application: **client** (front-end), **server** (back-end) and **deployment**.
```

Comment:
Should we include DB here too? Or is that implied in backend/server?

Reply:
I've copy pasted this part from before. I guess the intention was to emphasise **deployment** as a thing you get with Wasp.